p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -376,4 +376,13 @@ Layout Description
         <readOnly/>
         <ecc/>
     </section>
+    <section>
```

Comment:
this will be rebased behind the VERSION partition changes to make it secured; can you please update the HDAT partition documentation to say it's 32K instead of 16K as currently documented?